tools/clang/tools/dxcompiler/dxcdisassembler.cpp

```diff
@@ -347,6 +347,7 @@ PCSTR g_pFeatureInfoNames[] = {
     "64-bit Atomics on Heap Resources",
     "Advanced Texture Ops",
     "Writeable MSAA Textures",
+    "SampleCmpGradientOrBias",
```

Comment:
Need to add both flags at once, see: #6146.

Reply:
Added.

Reply:
I think we should be using the description strings from #6146:
```
    "SampleCmp with gradient or bias",
    "Extended command info",
```

Reply:
```suggestion
    "SampleCmp with gradient or bias",
```

Reply:
Note there are references to these in tests that will have to be fixed up as well, so it's not simply a matter of accepting this suggestion.

Reply:
Fixed.